laravel/app/Console/Commands/PushSchedule.php

```diff
@@ -91,10 +91,11 @@ public static function updateOption($date)
             $preference = Option::where('id', $option_id)->value('option');
             $milestones = Milestone::where('id', $milestone_id)->value('category');
             $objectives = Option::where('id', $option_id)->value('objectives');
+            //@$prefAnest = Option::where('id', $option_id)->value('anesthesiologist_name');
```

Comment:
not sure why there are changes to this file. it doesn't seem related to the functionality this PR should implement.